core/src/crds_gossip_pull.rs

```diff
@@ -391,25 +409,27 @@ impl CrdsGossipPull {
     // build a set of filters of the current crds table
     // num_filters - used to increase the likelyhood of a value in crds being added to some filter
     pub fn build_crds_filters(&self, crds: &Crds, bloom_size: usize) -> Vec<CrdsFilter> {
+        const PAR_MIN_LENGTH: usize = 1024;
```

Comment:
Maybe want to have some reasoning for this value.

Reply:
Just by experimenting using the added benchmark, I realized that there is no speed up for smaller arrays. It does not slow down the code, but I thought that probably better not to allocate threads for no gains.

Reply:
Ok, yea I mean is there any experiments that show 1024 is better than 512 or 256 or another constant.